app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -33,47 +32,66 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
 
   private val logger = Logger(this.getClass)
 
-  private val MovementSubmissionUrl = s"${appConfig.customsDeclareExportsMovements}${appConfig.saveMovementSubmission}"
-  private val MovementConsolidationUrl =
-    s"${appConfig.customsDeclareExportsMovements}${appConfig.submitMovementConsolidation}"
+  private val CustomsDeclareExportsMovementsUrl = s"${appConfig.customsDeclareExportsMovements}"
+  private val ArrivalSubmissionUrl =
```

Comment:
why `val`s from capital letter ?

Reply:
To indicate they're constants holding specific value.